sdk-extensions/jaeger-remote-sampler/src/main/java/io/opentelemetry/sdk/extension/trace/jaeger/sampler/JaegerRemoteSamplerProvider.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.extension.trace.jaeger.sampler;
+
+import io.opentelemetry.sdk.autoconfigure.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigurableSamplerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import java.util.concurrent.TimeUnit;
+
+public class JaegerRemoteSamplerProvider implements ConfigurableSamplerProvider {
+
+  @Override
+  public Sampler createSampler(ConfigProperties config) {
+    String serviceName = config.getString("jaeger.service");
+    JaegerRemoteSamplerBuilder builder = JaegerRemoteSampler.builder().setServiceName(serviceName);
+
+    // Optional configuration
+    String endpoint = config.getString("jaeger.remote.sampler.endpoint");
+    if (endpoint != null) {
+      builder.setEndpoint(endpoint);
+    }
+    Double initialRate = config.getDouble("jaeger.remote.sampler.initial.rate");
```

Comment:
I imagine we want this to be `initial_sampler` which can be set to any other possible value for `OTEL_SAMPLER`. Another good point we should go through the spec for

Reply:
removing this for now